### PR TITLE
#326 Team page redesigned

### PR DESCRIPTION
**Pull request by YuraRiabov**
Screenshot_903

### CODE REVIEW

**Review by Pavliuk13 — APPROVED** [review]

File: backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/TeamService.cs [hunk 1]
```diff
@@ -150,6 +150,13 @@ public async Task<ImagePathDto> UploadLogoAsync(IFormFile file, long? teamId)
         return new ImagePathDto() { Path = imagePath };
     }
 
+    public async Task DeleteLogo(long teamId)
+    {
+        var team = await GetTeamByIdAsync(teamId);
```
Comment (tatianahutii) [inline backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/TeamService.cs:155]: Cannot we delete files from storage as well?

File: frontend/src/app/modules/settings/team/team-preferences/team-preferences.component.ts [hunk 2]
```diff
@@ -134,6 +150,15 @@ export class TeamPreferencesComponent extends BaseComponent implements OnInit {
         control.patchValue(removeExcessiveSpaces(control.value));
     }
 
+    public deleteLogo() {
+        if (this.team?.id) {
+            this.teamService.deleteLogo(this.team.id)
+                .pipe(this.untilThis)
+                .subscribe();
+        }
+        this.imageUrl = '';
```
Comment (tatianahutii) [inline frontend/src/app/modules/settings/team/team-preferences/team-preferences.component.ts:170]: why not in subscribe?

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/app/modules/settings/team/team-preferences/team-preferences.component.ts (hunk 2 above)
Comment (YuraRiabov) [reply frontend/src/app/modules/settings/team/team-preferences/team-preferences.component.ts:170]: We need to set it as empty during team creation as well

Comment (tatianahutii) [reply frontend/src/app/modules/settings/team/team-preferences/team-preferences.component.ts:170]: how the deleteLogo method related to the team creation process?

**Review by tatianahutii — CHANGES_REQUESTED** [review]

**Review by tatianahutii — APPROVED** [review]

**Review by apofis96 — APPROVED** [review]

**Pull request merged**